ssl/ssl_cert.cc

```diff
@@ -489,6 +490,145 @@ UniquePtr<EVP_PKEY> ssl_cert_parse_pubkey(const CBS *in) {
   return UniquePtr<EVP_PKEY>(EVP_parse_public_key(&tbs_cert));
 }
 
+static long long asn1_utctime_to_unix_timestamp(const uint8_t *in,
```

Comment:
Hard to imagine this wouldn't already be present in BoringSSL?

Reply:
I avoided the ASN1 library since it was marked as being legacy [here](https://github.com/xvzcf/boringssl/blob/master/include/openssl/asn1.h#L75).